src/main.py

```diff
@@ -57,11 +65,11 @@ def save_metric_definitions(metric_names):
 
 
 def get_ltla_populations():
-    url = "https://www.ons.gov.uk/file?uri=/peoplepopulationandcommunity/populationandmigration/populationestimates" \
-          "/datasets/populationestimatesforukenglandandwalesscotlandandnorthernireland" \
-          "/mid2019april2019localauthoritydistrictcodes/ukmidyearestimates20192019ladcodes.xls"
+    url = "https://www.ons.gov.uk/file?uri=%2fpeoplepopulationandcommunity%2fpopulationandmigration%2fpopulationestimates" \
```

Comment:
This is moving to use more up to date population figures for Lower Tier Local Authorities, published [here](https://www.ons.gov.uk/peoplepopulationandcommunity/populationandmigration/populationestimates/datasets/populationestimatesforukenglandandwalesscotlandandnorthernireland). Motivated by the per 100,000 metric being slightly off what was shown on the [government dashboard](https://coronavirus.data.gov.uk/details/cases)

Reply:
nice fix